content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -235,11 +235,11 @@ You may install these packages in one of two ways: either via the NuGet Package
 
 `Install-Package Azure.Security.KeyVault.Secrets`
 
-### **Read Azure Key Vault Secrets in .NET Core**
+### Read Azure Key Vault Secrets in .NET Core
 
 In this section we’ll examine how to read secrets from AzureKeyVault.
 
-#### **Specify the Vault Uri in AppSettings**
+#### Specify the Vault Uri in AppSettings
 
 Create a section named KeyVault in the appsettings.json file and specify a key
```

Comment:
Wherever there is a file, please highlight it with ``

Here, `appsettings.json`

Reply:
Done, thanks!